src/appleseedmaya/shadingnode.cpp

```diff
@@ -447,16 +465,33 @@ MStatus ShadingNode::initialize()
                 MObject child2 = numAttrFn.create("uvFilterSizeY", "fsy", MFnNumericData::kFloat);
                 attr = numAttrFn.create("uvFilterSize", "fs", child1, child2);
             }
+            else if (p.mayaAttributeName == "outUV")
+            {
+                MObject child1 = numAttrFn.create("outU", "ou", MFnNumericData::kFloat);
+                MObject child2 = numAttrFn.create("outV", "ov", MFnNumericData::kFloat);
+                attr = numAttrFn.create("outUV", "o", child1, child2);
+            }
             else
             {
-                MObject child1 = numAttrFn.create(p.mayaAttributeName + "X", p.mayaAttributeShortName + "x", MFnNumericData::kFloat);
-                MObject child2 = numAttrFn.create(p.mayaAttributeName + "Y", p.mayaAttributeShortName + "y", MFnNumericData::kFloat);
+                MObject child1 = numAttrFn.create(
+                            p.mayaAttributeName + "X",
+                            p.mayaAttributeShortName + "x",
+                            MFnNumericData::kFloat,
+                            p.defaultValue.at(0));
+
+                MObject child2 = numAttrFn.create(
+                            p.mayaAttributeName + "Y",
+                            p.mayaAttributeShortName + "y",
+                            MFnNumericData::kFloat,
+                            p.defaultValue.at(1));
 
                 attr = numAttrFn.create(p.mayaAttributeName, p.mayaAttributeShortName, child1, child2);
+                CHECK_STATUS_AND_HANDLE_ERROR
             }
+
             if (!attr.isNull())
             {
-                status = AttributeUtils::makeInput(numAttrFn);
+                status = p.isOutput ? AttributeUtils::makeOutput(numAttrFn) : AttributeUtils::makeInput(numAttrFn);
```

Comment:
This is an issue in the original code, not your PR.
I think we should be using:
```
status = initializeAttribute(numAttrFn, p);
CHECK_STATUS_AND_HANDLE_ERROR
```
like all the other attribute types